VSCode/armsnippets.json

```diff
@@ -34,6 +58,15 @@
     ],
     "description": "ARM Parameter"
 },
+"Deployment Parameter": {
+    "prefix": "arm-deplparameter",
```

Comment:
Rename to "arm-paramvalue"? Also update description to say "parameter value"

Reply:
Way better name! Changing